### ISSUE TITLE
Pre-work Checklist: Developer: Suman

### ISSUE THREAD

**Issue by Suman2795** [hackforla/website#3553]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by kathrynsilvaconway** [comment]
@Suman2795 Hi there, are you still interested in working with Hack for LA? If you are, please leave a comment on this issue and attend one of our weekly dev team meetings so we get get you fully onboarded to our website team. Meetings are Tuesdays at 7pm pst and Sundays at 10am pst. Please let us know if you have any questions. If we do not hear back from you within 14 days, we will close this issue. Thanks!

**Comment by Suman2795** [comment]
Hi @kathrynsilvaconway! Unfortunately I was unavailable due to an emergency situation, but I will be there for the next meeting on Tuesday at 7pm. Thanks.

**Comment by Suman2795** [comment]
Hey @kathrynsilvaconway, I am working on my first issue and have made a pull request for it. I wanted to ask what are the next steps I need to do to review and merge it. Thanks!

**Comment by 7kram** [comment]
@Suman2795, congratulations on getting your good first and second issues merged 🙌. Please review the unchecked action items above and complete/check them off so that we can close this pre-work issue. 

We don't see that you're assigned to a small issue yet. Are you clear about what to do next?

Here are some links to the board sorted by size: small issues: [Frontend](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22size%3A+small%22+label%3A%22role%3A+front+end%22) and [Backend](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22size%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22) that you can choose from!

**Comment by Suman2795** [comment]
@7kram I have checked all the action items above for this issue to be closed and also assigned myself to a small issue.

**Comment by ExperimentsInHonesty** [comment]
He has an issue assigned to him that has not had an update since Dec 5th.  Sent him a message on the issue https://github.com/hackforla/website/issues/3598#issuecomment-1383209455

**Comment by ExperimentsInHonesty** [comment]
- I reopened issue https://github.com/hackforla/website/issues/3598 and am following up

**Comment by ExperimentsInHonesty** [comment]
@Suman2795 It's time to take up a large issue.
[large front end](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+large%22)
[large back end](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+back+end%2Fdevops%22)

In the meantime you seem to have the hang of the updates, so I am going to close this prework issue

**Comment by HackforLABot** [comment]
## Activity Log: Suman2795
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- Suman2795 opened: https://github.com/hackforla/website/issues/3553 at 09/13/2022, 07:54 PM PDT
- Suman2795 assigned: https://github.com/hackforla/website/issues/3553 at 09/13/2022, 07:55 PM PDT
- Suman2795 commented on: https://github.com/hackforla/website/issues/3553#issuecomment-1264676021 at 10/02/2022, 09:00 AM PDT
- Suman2795 assigned: https://github.com/hackforla/website/issues/2844#issuecomment-1048144299 at 10/07/2022, 06:30 AM PDT
- Suman2795 commented on: https://github.com/hackforla/website/issues/2844#issuecomment-1271608853 at 10/07/2022, 06:40 AM PDT
- Suman2795 opened: https://github.com/hackforla/website/pull/3600 at 10/07/2022, 08:23 AM PDT
- Suman2795 commented on: https://github.com/hackforla/website/issues/3553#issuecomment-1271855963 at 10/07/2022, 10:24 AM PDT
- Suman2795 PR was merged: https://github.com/hackforla/website/pull/3600#event-7550254365 at 10/09/2022, 06:18 PM PDT
- Suman2795 issue closed by PR 3600: https://github.com/hackforla/website/issues/2844#event-7550254434 at 10/09/2022, 06:18 PM PDT
- Suman2795 assigned: https://github.com/hackforla/website/issues/3202#issuecomment-1145813785 at 10/15/2022, 11:15 PM PDT
- Suman2795 commented on: https://github.com/hackforla/website/issues/3202#issuecomment-1279902000 at 10/15/2022, 11:20 PM PDT
- Suman2795 opened: https://github.com/hackforla/website/pull/3628 at 10/16/2022, 12:25 AM PDT
- Suman2795 commented on: https://github.com/hackforla/website/pull/3628#issuecomment-1280361887 at 10/16/2022, 11:37 PM PDT
- Suman2795 PR was merged: https://github.com/hackforla/website/pull/3628#event-7606459050 at 10/17/2022, 03:52 PM PDT
- Suman2795 issue closed by PR 3628: https://github.com/hackforla/website/issues/3202#event-7606459121 at 10/17/2022, 03:52 PM PDT
- Suman2795 assigned: https://github.com/hackforla/website/issues/2438#issuecomment-1180730183 at 10/19/2022, 09:22 AM PDT
- Suman2795 commented on: https://github.com/hackforla/website/issues/3553#issuecomment-1284299794 at 10/19/2022, 09:47 AM PDT
- Suman2795 commented on: https://github.com/hackforla/website/issues/2438#issuecomment-1284880247 at 10/19/2022, 08:46 PM PDT
- Suman2795 opened: https://github.com/hackforla/website/pull/3652 at 10/24/2022, 12:53 AM PDT
- Suman2795 assigned: https://github.com/hackforla/website/issues/2459#issuecomment-1292587475 at 10/29/2022, 06:05 AM PDT
- Suman2795 unassigned: https://github.com/hackforla/website/issues/2459#issuecomment-1295965027 at 10/29/2022, 01:30 PM PDT
- Suman2795 PR was merged: https://github.com/hackforla/website/pull/3652#event-7699613081 at 10/30/2022, 11:54 AM PDT
- Suman2795 issue closed by PR 3652: https://github.com/hackforla/website/issues/2438#event-7699613099 at 10/30/2022, 11:54 AM PDT
- Suman2795 assigned: https://github.com/hackforla/website/issues/3598 at 10/31/2022, 05:00 AM PDT
- Suman2795 commented on: https://github.com/hackforla/website/issues/3598#issuecomment-1298195826 at 11/01/2022, 01:15 AM PDT
- Suman2795 commented on: https://github.com/hackforla/website/pull/3695#issuecomment-1304839279 at 11/06/2022, 08:31 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3695#pullrequestreview-1169625815 at 11/06/2022, 08:52 AM PST
- Suman2795 commented on: https://github.com/hackforla/website/issues/3598#issuecomment-1311887041 at 11/11/2022, 08:00 AM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3718#issuecomment-1314796876 at 11/14/2022, 09:31 PM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3718#pullrequestreview-1180675345 at 11/15/2022, 03:07 AM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3719#issuecomment-1316337272 at 11/15/2022, 08:45 PM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3719#pullrequestreview-1182984829 at 11/16/2022, 09:02 AM PST
- Suman2795 commented on: https://github.com/hackforla/website/issues/3598#issuecomment-1319468705 at 11/17/2022, 06:32 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3729#issuecomment-1319469331 at 11/17/2022, 06:34 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3730#issuecomment-1319962080 at 11/18/2022, 04:58 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3729#pullrequestreview-1186419399 at 11/18/2022, 08:33 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3730#pullrequestreview-1186509014 at 11/18/2022, 09:26 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3730#pullrequestreview-1186511738 at 11/18/2022, 09:27 AM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3737#issuecomment-1323105967 at 11/21/2022, 09:35 PM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3737#pullrequestreview-1191758758 at 11/23/2022, 06:32 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3751#pullrequestreview-1196260044 at 11/28/2022, 09:31 AM PST
- Suman2795 issue closed as completed: https://github.com/hackforla/website/issues/3553#event-7913448608 at 11/29/2022, 06:57 AM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3764#issuecomment-1336431015 at 12/04/2022, 06:49 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3764#pullrequestreview-1205700744 at 12/05/2022, 07:05 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/issues/3598#issuecomment-1338676642 at 12/05/2022, 07:12 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3816#issuecomment-1383595330 at 01/15/2023, 11:22 PM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3816#pullrequestreview-1249687384 at 01/15/2023, 11:49 PM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3816#pullrequestreview-1249687829 at 01/15/2023, 11:50 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/issues/3598#issuecomment-1396449359 at 01/18/2023, 09:12 PM PST
- Suman2795 opened: https://github.com/hackforla/website/issues/3843 at 01/18/2023, 11:43 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/issues/3843#issuecomment-1397926952 at 01/19/2023, 09:05 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/3935#issuecomment-1417774543 at 02/05/2023, 05:00 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/3935#pullrequestreview-1284305266 at 02/05/2023, 09:28 AM PST
- Suman2795 issue closed as completed: https://github.com/hackforla/website/issues/3553#event-8609025922 at 02/25/2023, 10:59 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/4056#issuecomment-1450158145 at 03/01/2023, 05:30 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/4056#pullrequestreview-1321212761 at 03/02/2023, 12:04 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/4056#pullrequestreview-1323201989 at 03/02/2023, 11:13 PM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/4099#issuecomment-1454784299 at 03/04/2023, 07:53 AM PST
- Suman2795 submitted review: https://github.com/hackforla/website/pull/4107#pullrequestreview-1328471755 at 03/07/2023, 05:30 AM PST
- Suman2795 commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1475277466 at 03/19/2023, 07:32 AM PDT
- Suman2795 submitted review: https://github.com/hackforla/website/pull/4197#pullrequestreview-1350775585 at 03/21/2023, 08:32 AM PDT
- Suman2795 submitted review: https://github.com/hackforla/website/pull/4197#pullrequestreview-1353969153 at 03/23/2023, 12:09 AM PDT
